tests/integration/fix-364.spec.js

```diff
@@ -0,0 +1,57 @@
+import { cy, describe, before, it } from 'local-cypress';
+
+describe('Issue #364', function () {
+  before(() => {
+    cy.viewport('macbook-15');
+  });
+
+  beforeEach(() => {
+    cy.visit('/#/dz60/LAYOUT');
+
+    cy.intercept('POST', 'https://api.qmk.fm/v1/compile', { fixture: '364-first-step.json' });
+    cy.intercept('GET', 'https://api.qmk.fm/v1/compile/ea1514b3-bdfc-4a7b-9b5c-08752684f7f6', 
+      { fixture: '364-second-step.json' });
+  });
+
+  it('"download firmware" button should be enabled after pressing compile', function () {
+    cy.wait(1000);
+
+    cy.get('#compile', { timeout: 5000 }).click();
```

Comment:
does this test even work? compiles can take minutes.

Reply:
This test uses feature called 'fixtures' (API mocking). 

When configurator sends request POST `https://api.qmk.fm/v1/compile` cypress sends response from `364-first-step.json` and when applications sends GET `https://api.qmk.fm/v1/compile/{id}` then cypress sends response from `364-second-step.json`.

Reply:
I see. thanks for clarifying. It's not a feature I've used. Very neat